recipes/qtpyinheritance/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "qtpyinheritance" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/klauer/qtpyinheritance/archive/v{{ version }}.tar.gz
+  sha256: 1a2194d75d49d2ec1839ab0db786c322ca67131109377f61419372b7ffd0f372
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - qtpy
+
+test:
+  imports:
+    - qtpyinheritance
+  requires:
+    - pyqt
```

Comment:
This isn't needed as it's already in the `run` depnendencies:
```suggestion
```

Reply:
`qtpy` is in the run dependencies, which generically supports either pyqt or pyside, but will fail on import without any bindings installed. That means to test the import we'd need pyqt, for example.

```
$ python -c 'import qtpy'
...

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/klauer/mc/envs/testqt/lib/python3.7/site-packages/qtpy/__init__.py", line 210, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
```

Reply:
As `pyqt` is in `run` this line does nothing

Reply:
`qtpy` is in run
`pyqt` is in test

Am I misunderstanding something... ?

Reply:
No, sorry I just can't read 😆

Reply:
No problem - they're easy to confuse!